### PR TITLE
fix build by providing options to vulcanize correctly

### PR DESCRIPTION
**Pull request by azakus**

### CODE REVIEW

**Comment by azakus** [comment]
PTAL @garlicnation, @justinfagnani

File: src/build/optimize.ts [hunk 1]
```diff
@@ -26,31 +26,30 @@ export interface OptimizeOptions {
...
-    removeComments?: boolean;
+    [property: string]: any;
   };
   /**
    * Enable CSS minification
    */
-  css?: {
-    stripWhitespace?: boolean;
+  css?: boolean | {
+    [property: string]: any;
```
Comment (justinfagnani) [inline src/build/optimize.ts]: Why the union type? DOesn't look like you're using the object variant.

**Comment by justinfagnani** [comment]
LGTM

File: src/build/optimize.ts (hunk 1 above)
Comment (azakus) [inline src/build/optimize.ts]: The properties in there aren't even used, so I'm not 100% sure what the interface should be anymore. That should probably not be in _this_ PR though.

**Pull request merged**